internal/handler/web/studies.go

```diff
@@ -48,6 +49,32 @@ func studyToOpenApiStudy(study types.Study) openapi.Study {
 	}
 }
 
+func assetToOpenApiAsset(asset types.Asset) openapi.Asset {
+	AssetUserIDStr := asset.UserID.String()
+	AssetStudyIDStr := asset.StudyID.String()
+
+	return openapi.Asset{
+		Id:                     asset.ID.String(),
+		UserId:                 &AssetUserIDStr,
+		StudyId:                &AssetStudyIDStr,
```

Comment:
these can be required fields(?)

Reply:
https://github.com/ucl-arc-tre/portal/pull/250/commits/1543375b77cbc2be972b7e514b09c7cc0a2d8916